### ISSUE TITLE
Tooltip hotspots

### ISSUE THREAD

**Issue by aguilar1181** [timmywil/jquery.panzoom#195]
is there anyway to have this working with tooltips? I'm trying to build a mall map that has the capabilities of panzoom but tooltips at the sametime

**Comment by ilokhov** [comment]
I'm currently working on something similar - building an interactive plan where the user can click/hover to get additional info on various objects.

You can define SVG elements that will act as your tooltip messages as being hidden by default - for example assign a class which has display: none; in it to them. Then use jQuery mouseenter and mouseleave or hover functions to remove the class of those elements when your object is hovered on.

**Comment by EvilDonkeyLondon** [comment]
Hi, any news on the tooltips functionality?

I have absolute positioned in the panzoom container some elements that function as tooltip icons. They work fine but when I resize the page they don't sit in the right place. Is there a way of having elements stay always in place within panzoom?

**Comment by ilokhov** [comment]
How did you insert your tooltip icons? Are they HTML elements or did you insert them using SVG? If you use SVG it should function well together with panzoom.

**Comment by EvilDonkeyLondon** [comment]
I added them in as HTML elements, I'll give it a go with SVGs then. Thank you!

**Comment by timmywil** [comment]
Tooltips will work fine as long as there aren't conflicting event handlers. If there are, it can be difficult to ensure the right plugin is given priority. Would need a test case to help further.

**Comment by mhulse** [comment]
Hi all,

Apologies for resurrecting a closed thread …

I was just curious if anyone would like to share an example of the pan/scroll zoom project they worked on.

I am tasked with making a pannable/scrollable/zoomable "work" area, and I'm trying to learn the best way to do this with the jquery.panzoom plugin.

I'm currently discovering that absolute positioning is tricky when an element goes outside of the canvas. Now I'm playing with css3 `translation` and some JS to make the panzoom container grow as new elements get added/removed.

Long story short, I'm really curious to see how others have pulled off working with child elements that are more complex than a simple image?
